### PR TITLE
Addition of new segments needed for CP Hunters

### PR DESCRIPTION
**Pull request by LatiusAuro**
Addition of the alternative invalid map segment, and including the start of the segments for the CP Hunters text.

### CODE REVIEW

File: Media/Translations/Default.xml [hunk 1]
```diff
@@ -148,6 +148,19 @@
 		<ru>Невозможно загрузить карту, переход к следующей.</ru>
 		<sk>Táto mapa nie je platná, prechádzam na ďalšiu.</sk>
 	</segment>
+	<segment>
```
Comment (domino54) [inline Media/Translations/Default.xml]: A segment with identical meaning already exists. It looks like this:
> This map is invalid; moving to the next one.
Please remove this segment and use the mentioned one in your code.

File: Media/Translations/Default.xml [hunk 2]
```diff
@@ -2899,4 +2912,28 @@
 		<nl>brandstof</nl>
 		<pl>Paliwo</pl>
 	</segment>
-</resources>
\ No newline at end of file
+	<segment>
```
Comment (domino54) [inline Media/Translations/Default.xml:2902]: Please use the words "Checkpoint" and "Checkpoints" for better understanding for inexperienced players.

**Review by domino54 — CHANGES_REQUESTED** [review]

File: Media/Translations/Default.xml (hunk 2 above)
Comment (LatiusAuro) [reply Media/Translations/Default.xml:2902]: Done

**Comment by LatiusAuro** [comment]
Changes have been made. "This map is invalid; moving to the next one." is now used, and CP/CPs are changed to Checkpoint/Checkpoints.

**Review by domino54 — APPROVED** [review]

**Comment by domino54** [comment]
Thanks!

**Pull request merged**